common/libs/network_protocol/network_protocol.proto

```diff
@@ -0,0 +1,130 @@
+syntax = "proto2";
```

Comment:
We should clang-format this file.  Happy to do it myself, but if it's easy for you to do, then you can avoid merge conflicts later...

```
clang-format -i network_protocol.proto
```

and modify .pre-commit-config.yaml to add a new clang-format rule.

Reply:
I have no idea how to update the yaml file, maybe you can do this?

I have formatted the file though.

Reply:
Thanks for formatting it.  Sent YAML change in https://github.com/RespiraWorks/VentilatorSoftware/pull/133